public/css/style.css

```diff
@@ -145,9 +145,11 @@ section {
     font-size: 12px;
     text-align: center;
     padding: 30px 0;
+    /* position: relative; */
     position: relative;
     margin-top: -3em;
     width: 100%;
+    bottom: 0;
```

Comment:
There should not be any difference after adding this line, so please remove it unless you see any noticeable difference. The footer problem I explained in the comments of the sponsor file.